### PR TITLE
fix text-align on team

### PR DESCRIPTION
**Pull request by hybes**
## What does this PR do?

Fix text alignment for long team namess

Fixes # (issue)

Before:
without

After:
with

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

- [ ] Test A
- [ ] Test B

## Checklist

- My code doesn't follow the style guidelines of this project
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my PR needs changes to the documentation
- I haven't checked if new and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by CarinaWolli** [comment]
@Hybes Could you also fix the cut-off team name? We should make sure to not cut off any of the gray hover background

It currently looks like that: 
Screenshot 2023-02-24 at 11 01 24

**Comment by Hybes** [comment]
> @Hybes Could you also fix the cut-off team name? We should make sure to not cut off any of the gray hover background
> 
> It currently looks like that: Screenshot 2023-02-24 at 11 01 24

Yup, I can add that

**Comment by Hybes** [comment]
@CarinaWolli can you try now, with this commit. should do exactly as requested.

**Review by CarinaWolli — COMMENTED** [review]
That works! Found one more issue:
Screenshot 2023-02-24 at 11 59 31
In that case we could just truncate the long word with an ellipsis

**Comment by Hybes** [comment]
@CarinaWolli all done, try now - let me know if you think of any more

File: packages/features/settings/layouts/SettingsLayout.tsx [hunk 1]
```diff
@@ -261,7 +261,10 @@ const SettingsSidebarContainer = ({
                                 className="h-[16px] w-[16px] self-start rounded-full stroke-[2px] ltr:mr-2 rtl:ml-2 md:mt-0"
                                 alt={team.name || "Team logo"}
                               />
-                              <p>{team.name}</p>
+                              <p
+                              className="w-1/2 truncate"
+                              >
```
Comment (zomars) [inline packages/features/settings/layouts/SettingsLayout.tsx:266]: Why is this on  3 lines?

Comment (Hybes) [reply packages/features/settings/layouts/SettingsLayout.tsx:266]: @zomars probs some Prettier shit, you want it on one?

File: packages/features/settings/layouts/SettingsLayout.tsx [hunk 2]
```diff
@@ -261,7 +261,10 @@ const SettingsSidebarContainer = ({
...
+                                {team.name}</p>
```
Comment (zomars) [inline packages/features/settings/layouts/SettingsLayout.tsx:267]:
```suggestion
                              <p className="w-1/2 truncate">{team.name}</p>
```

**Review by zomars — APPROVED** [review]

**Comment by zomars** [comment]
Thank you for your contribution 🙏

**Pull request merged**